plugins/techdocs/src/reader/transformers/addGitFeedbackLink.ts

```diff
@@ -52,7 +52,10 @@ export const addGitFeedbackLink = (
       `Page source:\n${sourceAnchor.href}\n\nFeedback:`,
     );
     const gitInfo = parseGitUrl(sourceURL.pathname);
-    const repoPath = `/${gitInfo.organization}/${gitInfo.name}`;
+    const repoPath =
+      integration?.type === 'github'
+        ? sourceURL.pathname.split('/').slice(0, 3).join('/')
```

Comment:
Just an idea: How about applying some kind of reverse-transformation of the url to replace the `edit` back to a `blob` or `tree` and keep the git-parse-url functionality intact? (see [GitHubIntegration](https://github.com/backstage/backstage/blob/4fb3a7ada4bd1fccead0208a3b3459bf76aa54e2/packages/integration/src/github/GitHubIntegration.ts#L60-L74))

Reply:
ah, that works too, thanks!